Yosemite/YosemiteTests/Mocks/Networking/Remote/MockAccountRemote.swift

```diff
@@ -98,12 +98,18 @@ extension MockAccountRemote: AccountRemoteProtocol {
         // no-op
     }
 
-    func loadUsernameSuggestions(from text: String) async -> Result<[String], Error> {
+    func loadUsernameSuggestions(from text: String) async throws -> [String] {
         guard let result = loadUsernameSuggestionsResult else {
             XCTFail("Could not find result for loading username suggestions.")
-            return .failure(NetworkError.notFound)
+            throw NetworkError.notFound
+        }
+
+        switch result {
+        case let .success(suggestions):
+            return suggestions
+        case let .failure(error):
+            throw error
```

Comment:
`Result.get()` is a convenient way to convert from result to throwing:

```suggestion
        return try result.get()
```

I see a couple other instances of this in YosemiteTests

Reply:
True, done in 1a62a27e3ad8a9a3c5c42fa9f8786d28f026c6bb